modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2IDTokenEncryptionTestCase.java

```diff
@@ -458,8 +464,8 @@ private void initServiceProviderKeys() throws Exception {
      * @param client         CloseableHttpClient object to send the login post.
      * @param sessionDataKey String sessionDataKey obtained.
      * @return Extracted sessionDataKeyConsent.
-     * @throws IOException
-     * @throws URISyntaxException
+     * @throws IOException Error
+     * @throws URISyntaxException Error
```

Comment:
Missing fullstops at the end

Reply:
Fixed with https://github.com/wso2/product-is/pull/20096